cvmfs/server/cvmfs_server_common.sh

```diff
@@ -1306,4 +1307,34 @@ _run_catalog_migration() {
   set_ro_root_hash $name $trunk_hash || die "Root hash update failed";
 }
 
+upload_statistics_plots_bg() {
+  local db_path_copied="$1"
+  local REPO_NAME="$2"
+  local upstream="$3"
+  root_file=$(mktemp -t stats.root.XXXXXX)
+  root -l -b -q /usr/share/cvmfs-server/generate_stats_plots.C+\("\"${db_path_copied}\", \"${root_file}\""\)
```

Comment:
Apologies, this should be really the last fix to this PR: Let's gracefully fail if ROOT is not available, i.e. print an error message explaining to the user that ROOT is missing.